File: Soundbox.Reloaded.Ui.Presentation/Dto/Sounds/SoundDto.cs
using System;

namespace Soundbox.Reloaded.Ui.Presentation.Dto.Sounds
{
    public class SoundDto : EntityBaseDto
    {
        public Guid CategoryId { get; set; }

        public string CategoryName { get; set; }

        public string Title { get; set; }

        public string ArtistName { get; set; }

        public string SoundFileName { get; set; }

        public string ImageFileName { get; set; }
    }
}

File: Soundbox.Reloaded.Ui.Web/scripts/domain/sounds/SoundDto.ts
module soundbox.domain.sounds {
    export class SoundDto extends EntityBaseDto {
        public CategoryId: string;
        public CategoryName: string;
        public Title: string;
        public ArtistName: string;
        public SoundFileName: string;
        public ImageFileName: string;

        public imageFilePathOrDefault(): string {
            return this.ImageFileName == null || this.ImageFileName == ""
                 ? '/assets/default-sound.png'
                 : '/ImageFiles/' + this.ImageFileName;
        }
    }
}